src/NzbDrone.Core/IndexerSearch/EpisodeSearchService.cs

```diff
@@ -66,6 +66,41 @@ public void MissingEpisodesAiredAfter(DateTime dateTime, IEnumerable<Int32> grab
             _logger.ProgressInfo("Completed search for {0} episodes. {1} reports downloaded.", missing.Count, downloadedCount);
         }
 
+        private void SearchForMissingEpisodes(List<Episode> episodes)
+        {
+            _logger.ProgressInfo("Performing missing search for {0} episodes", episodes.Count);
+            var downloadedCount = 0;
+
+            using (var rateGate = new RateGate(100, TimeSpan.FromSeconds(60)))
```

Comment:
this assumes we can process ~2 items a second, thats going to all the indexers and processing the results in 600ms.

also, why would we rate block this? it's happening serially not in parallel.